client/src/styles/global.css

```diff
@@ -0,0 +1,53 @@
+
+/* global.css */
```

Comment:
No need to write the name of the file here

Reply:
Done